docs/create/cloudrun/index-cloudrun.mdx

```diff
@@ -0,0 +1,282 @@
+
+Allow unauthenticated invocations to [google-cloud-run] (y/N)?  y
+
+Building using Dockerfile and deploying container to Cloud Run service [google-cloud-run] in project [redislabs-marketing-project] region [asia-east1]
+⠼ Building and deploying new service... Uploading sources.
+  ✓ Creating Container Repository...
+  ✓ Uploading sources...
+  . Building Container...
+  . Creating Revision...
+  . Routing traffic...
+  . Setting IAM Policy…
+
+Please enter your numeric choice:  1
+
+To make this the default region, run `gcloud config set run/region asia-east1`.
+
+Allow unauthenticated invocations to [django-redis-rate-limiting-example] (y/N)?  y
+
+Deploying container to Cloud Run service [django-redis-rate-limiting-example] in project [redislabs-marketing-project] region [asia-east1]
+✓ Deploying new service... Done.
+  ✓ Creating Revision... Revision deployment finished. Waiting for health check to begin.
+  ✓ Routing traffic...
+  ✓ Setting IAM Policy...
+Done.
+Service [django-redis-rate-limiting-example] revision [django-redis-rate-limiting-example-00001-mog] has been deployed and is serving 100 percent of traffic.
+Service URL: https://django-redis-rate-limiting-example-opcboau66a-de.a.run.app
+
+
+```
+
+
+
+### Step 6. Verify if the service is listed under Cloud Run dashboard
+
+
+
+
+![alt_text](images/gcloud1.png "image_tooltip")
+
+
+
+![alt_text](images/gcloud2.png "image_tooltip")
+
+
+#### Verifying the service via terminal UI
+
+
+```
+$ gcloud run services describe django-redis-rate-limiting-example
+
+✔ Service django-redis-rate-limiting-example in region asia-east1
+
+URL:     https://django-redis-rate-limiting-example-opcboau66a-de.a.run.app
+Ingress: all
+Traffic:
+  100% LATEST (currently django-redis-rate-limiting-example-00001-mog)
+
+Last updated on 2022-02-19T02:50:03.100357Z by ajeet.raina@redis.com:
+  Revision django-redis-rate-limiting-example-00001-mog
+  Image:           gcr.io/redislabs-marketing-project/django-redis-rate-limiting-example@sha256:95bf8d5705c4ab52d2c9ba07d7dcf0c651431d8d5d38ad8806487aaa8a8a870d at 95bf8d57...
+  Port:            8080
+  Memory:          512Mi
+  CPU:             1000m
+  Service account: 406459833831-compute@developer.gserviceaccount.com
+  Concurrency:     80
+  Max Instances:   100
+  Timeout:         300s
+
+```
+
+
+### Step 7. Set up environment variables 
+
+
+
+- Adding Redis URL to point to Redis Enterprise Cloud
+
+
+
+
+![alt_text](images/gcloud3.png "image_tooltip")
+
+
+
+
+- Enable HTTPS
+
+
+
+
+![alt_text](images/gcloud4.png "image_tooltip")
+
+
+
+-  Ensuring the VPC connector shows as “none”
+
+
+
+![alt_text](images/gcloud5.png "image_tooltip")
+
+
+
+- Allowing all traffic
+
+
+
+![alt_text](images/gcloud6.png "image_tooltip")
+
+
+### Step 8. Access your app
+
+Open [https://django-redis-rate-limiting-example-opcboau66a-de.a.run.app/](https://django-redis-rate-limiting-example-opcboau66a-de.a.run.app/) and access your app.
```

Comment:
@ajeetraina maybe needs a sentence or two explaining where this URL comes from and that the URL in the article here is just an example, your URL will differ if you are following this tutorial?